packages/utilities/psammead-styles/src/typography.jsx

```diff
@@ -0,0 +1,17 @@
+import { shape, func } from 'prop-types';
+import styled from 'styled-components';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const TypographyText = styled.p`
+  ${props =>
+    props.script && props.typographyFunc
+      ? props.typographyFunc(props.script)
+      : ''}
+`;
+
+TypographyText.propTypes = {
```

Comment:
After moving it. Here we can safely ignore the proptype checks with
`/* eslint-disable react/prop-types */`